src/main/java/de/tum/in/www1/artemis/repository/ComplaintRepository.java

```diff
@@ -24,6 +24,9 @@
 @Repository
 public interface ComplaintRepository extends JpaRepository<Complaint, Long> {
 
+    @Query("SELECT c FROM Complaint c LEFT JOIN c.result r LEFT JOIN r.submission s WHERE s.id = :#{#submissionId}")
```

Comment:
We try to have a style convention when it comes to db queries, see https://docs.artemis.ase.in.tum.de/dev/guidelines/server/#sql-statement-formatting

Reply:
Thanks, I just replaced it. Good catch!

Reply:
I fixed the style and also adapted it for some other definitions in the same file.

Reply:
Thanks :)